src/Resources/Resources/ChangeLog.md

```diff
@@ -22,6 +22,9 @@
     - Add `-SkipResourceNameParameterization` parameter to Export-AzResourceGroup
     - Add `-SkipAllParameterization` parameter to Export-AzResourceGroup
     - Add `-Resource` parameter to Export-AzResourceGroup for exported resource filtering
+    - Fix help text for Get-AzPolicyState -Top parameter
```

Comment:
@mentat9 we just had a release, so there shouldn't be anything existing under the `## Upcoming Release` header -- I would pull the latest from the `master` branch and ensure that the only content under the `## Upcoming Release` header is the lines you are adding

Reply:
Updated.